core-web/libs/portlets/dot-languages/portlet/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "target": "es2022",
+        "useDefineForClassFields": false,
+        "forceConsistentCasingInFileNames": true,
+        "strict": true,
+        "noImplicitOverride": true,
+        "noPropertyAccessFromIndexSignature": true,
+        "noImplicitReturns": true,
+        "noFallthroughCasesInSwitch": true
```

Comment:
Doesn't this overwrite the `compilerOptions` of the `tsconfig.base.json`?

Reply:
not really, the only `target` is defined in the `tsconfig.base.json`